analyzers/src/SonarAnalyzer.CFG/SonarAnalyzer.CFG.csproj

```diff
@@ -40,6 +40,12 @@
     <NoWarn>NU1605</NoWarn>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
It is possible to only add this PackageReference once in `Directory.Build.props`, however:
- this leads to a problem with assembly naming between `SonarAnalyzer.Common` and `SonarAnalyzer.Vsix` (both have AssemblyName `SonarAnalyzer`) - and I didn't want to waste time on that as it's not necessary
- as an initial baby step, I believe it's ok to use this on the main projects and we can change things as we go, after dogfooding what we have here

Reply:
Did you consider a shared prop file for this?

Reply:
I haven't, and I should

Reply:
Note for @pavel-mikula-sonarsource - I name the shared props file `.targets` to respect the convention suggested by Microsoft [here](https://docs.microsoft.com/en-us/visualstudio/msbuild/import-element-msbuild?view=vs-2019#remarks)